src/Modules/Admin/Commands.Admin/Microsoft.PowerBI.Commands.Admin.format.ps1xml

```diff
@@ -27,5 +27,70 @@
         </ListEntries>
       </ListControl>
     </View>
+    <View>
+      <Name>Microsoft.PowerBI.Common.Api.Workspaces.Workspace</Name>
+      <ViewSelectedBy>
+        <TypeName>Microsoft.PowerBI.Common.Api.Workspaces.Workspace</TypeName>
+      </ViewSelectedBy>
+      <ListControl>
+        <ListEntries>
+          <ListEntry>
+            <ListItems>
+              <ListItem>
+                <Label>Id</Label>
+                <ScriptBlock>$_.Id</ScriptBlock>
+              </ListItem>
+              <ListItem>
+                <Label>Name</Label>
+                <ScriptBlock>$_.Name</ScriptBlock>
+              </ListItem>
+              <ListItem>
+                <Label>IsOnDedicatedCapacity</Label>
+                <ScriptBlock>$_.IsOnDedicatedCapacity</ScriptBlock>
+              </ListItem>
+              <ListItem>
+                <Label>CapacityId</Label>
+                <ScriptBlock>$_.CapacityId</ScriptBlock>
+              </ListItem>
+              <ListItem>
```

Comment:
Merge this into Workspace ps1xml based on my above feedback

Reply:
Merged